Fill.py

```diff
@@ -529,7 +548,7 @@ def failed(warning: str, force):
                 for key, value in locations.items():
                     location_list += [key] * value
                 locations = location_list
-            if isinstance(locations, str):
+            if isinstance(locations, str):  # TODO: isn't this already handled 8 lines up?
```

Comment:
@Berserker66?

Reply:
Yes, it's also a bad way of doing it. the "location" block should just create a list, instead of checking later.